src/notation/view/widgets/breaksdialog.cpp

```diff
@@ -37,6 +37,16 @@ BreaksDialog::BreaksDialog(QWidget* parent)
     setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
     intervalButton->setChecked(true);
     intervalBox->setValue(DEFAULT_INTERVAL);
+
+    //: `%1` will be replaced with a number input field.
+    //: Text before it will appear before that number field, text after will appear after the field.
+    QString text = qtrc("notation/add-remove-system-breaks", "Break systems every %1 measures");
+    QRegularExpression regex("(.*)%1(.*)");
```

Comment:
Why do we use regexp here?

Reply:
Actually this one is simple enough to use a simple `.split("%1")`. Will do!